### PR TITLE
Migrations path

### PR DESCRIPTION
**Pull request by Lehersun**

### CODE REVIEW

File: src/functions.php [hunk 1]
```diff
@@ -25,5 +25,5 @@ function getDataArray(object $obj): array
  */
 function getMigrationsPath(): string
 {
-    return __DIR__ . '\..\tests\lib\Migrations';
+    return __DIR__ . DIRECTORY_SEPARATOR . '..' . '/tests/lib/Migrations';
```
Comment (Magomogo) [inline src/functions.php]: If you want to support windows, you need this:

    return __DIR__ . DIRECTORY_SEPARATOR . '..' . DIRECTORY_SEPARATOR . 'tests' . DIRECTORY_SEPARATOR . 'lib' . DIRECTORY_SEPARATOR . 'Migrations';


Also please consider using `realpath()` function to eliminate `..` part.

File: tests/integration/GlobalFunctionsTest.php [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+namespace GroupLife\Core\tests\integration;
+
+use PHPUnit\Framework\TestCase;
+
+class GlobalFunctionsTest extends TestCase
+{
+    public function testGetMigrationsPath()
+    {
+        self::assertFileExists(getMigrationsPath() . '/Version20210913225128.php');
```
Comment (Magomogo) [inline tests/integration/GlobalFunctionsTest.php]: Here you need a directory separator too. Or move the ending slash to `getMigrationsPath()` method.

**Review by Magomogo — CHANGES_REQUESTED** [review]

**Comment by Lehersun** [comment]
Nice function, added to the arsenal. Thanks

**Comment by Lehersun** [comment]
should we correct namespace of the global functions too?

**Pull request merged**